bench/wtperf/wtperf.c

```diff
@@ -762,6 +762,90 @@ worker(void *arg)
 					value_buf[0] = 'a';
 				if (opts->random_value)
 					randomize_value(thread, value_buf);
+
+				/*
+				 * Distribute the modifications across
+				 * whole document
+				 */
+				if (opts->distribute_modify)
+				{
+					int nentries = 16;
+					size_t iter;
+					size_t total_modify_size;
+					size_t modify_size;
+					size_t modify_offset;
+
+					/* 10% of total size of the record */
+					total_modify_size = strlen(value) / 10;
+
+					/*
+					 * maximum modification size per
+					 * modify
+					 */
+					modify_size = (total_modify_size
+							/ (size_t)nentries);
```

Comment:
Continuation lines are indented 4 spaces.

Reply:
Most of the indentation issues are fixed (maybe some are not proper as the coding guidelines)